src/js/shield_defs.js

```diff
@@ -4201,6 +4204,33 @@ export function loadShields() {
 
   // Ref-specific cases. Each entry should be documented in CONTRIBUTE.md
 
+  shields["CA:NS:S"].overrideByRef = {
+    BdOLSD: {
```

Comment:
Do people or publications normally refer to the routes by these initialisms? If so, let’s add an explanation to CONTRIBUTING.md, in the section where we justify other unsignposted refs.

Reply:
I don't think so, I just made up the initialisms so we'd have something to key off of. I don't think we can key off the route name because I thiiiiink the name on the tile comes only from the way itself. The Merritt Parkway is the only one we do like that so far and it happens to have all way names match the parkway relation name.

This situation is kind of like the Great Lakes Circle Tour routes, but with a more items and less documentation. I imagine the problem will come up more often as we add additional networks of scenic routes.

I can add documentation for this.

Reply:
For #451, the New Zealand community apparently decided to give each route a distinct `network` subvalue in order to avoid polluting `ref` with unsignposted initialisms. However, this fragmentation makes less sense when the shield designs all follow a common motif, as in this case.

I think inevitably we’ll need the tiles to contain something beyond the `network=ref` syntax in `route_*` properties. If we could go back and change the format, I would’ve suggested something like `route:2:network` or `route_network_2`, which would’ve also logically allowed for `route_name_2` or even `route_wikidata_2` for these tricky cases.

Reply:
> This situation is kind of like the Great Lakes Circle Tour routes, but with a more items and less documentation.

The Great Lakes Circle Tour routes are known by the initialisms in `ref`, at least informally, so it isn’t a big deal if a non-shield-capable renderer displays the `ref` verbatim. I’d still reach out to the local community to get their opinion on this use of `ref`, in case there are more established initialisms for the routes.

Reply:
> Regarding the choice of ref on bespoke tourist routes, don't we have `overrideByName` as an option? Then we would be tied to the name, which should be beyond dispute. In any case, the choice of `ref` value on the route relation of an obscure tourist route seems like something we can just make up using reasonable conventions and change later if the community changes their mind down the line. In short, I'd favor pushing this forward either as-is or with a name-based definition.

@ZeLonewolf `overrideByName` was introduced for the Kentucky parkways. I agree that it would be an ideal solution… except that the `name` is just the way name, not the route relation’s name. For tourist routes, the road name isn’t reliably the route relation’s name, although it is in the case of the Cabot Trail.

Reply:
Yep, here's an illustrative example. Names like "Lighthouse Route" don't always appear in the data, so we're left to rely on a `ref` at the moment.

<img width="266" alt="Screenshot 2023-08-14 at 1 54 06 PM" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/2046746/c257d748-8ba9-4bae-8c5b-b50ebfd00db5">

Reply:
Yep, makes sense to me - thanks for refreshing how that works. I built the damn logic for those `route_X` attributes and I can't even keep it straight...

Reply:
Let’s ticket out something about a `route_#_name` scheme in OpenMapTiles. Something similar would also help with Spain’s per-route colors in #654.

Reply:
As of #1032, each feature has a `route_#_name` property based on the route relation. The name gets plumbed through to the shield selection code, so you should be able to specify `overrideByName` without worrying about the roadway carrying a different name, and the relations no longer need to be tagged with initialisms as `ref`s.

<img src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/1231218/9ea42d6f-892c-43b9-9777-b0ad7a171bbd" width="214" alt="route_1_network: CA:NS:R; route_1_ref: 322; route_1_name: Route 322; route_2_network: CA:NS:S; route_2_name: Lighthouse Route">